### DISCUSSION TITLE
Manifest unknown for bitwardenrs/server:raspberry

### DISCUSSION THREAD

**Discussion by 1011001-p** [dani-garcia/vaultwarden#1364] [Q&A]
### Subject of the issue

Using my Raspberry Pi 4 (4GB) (Raspbian GNU/Linux 10 (buster) armv7l) and docker (Docker version 20.10.3, build 48d30b5) I tried to install bitwarden_rs with the following command:
docker pull bitwardenrs/server:raspberry
I found this command in many guides around the internet but for some reason it fails with the following error:
Error response from daemon: manifest for bitwardenrs/server:raspberry not found: manifest unknown: manifest unknown
I checked docker hub and I cannot find the tag "raspberry" so I think it was removed maybe?

### Your environment

* Bitwarden_rs version: bitwardenrs/server:raspberry

* Install method: docker pull bitwardenrs/server:raspberry
* Clients used: 
* Reverse proxy and version: 
* Version of mysql/postgresql: 
* Other relevant information: Running on Raspberry Pi 4 (4 GB of RAM model)

### Steps to reproduce

docker pull bitwardenrs/server:raspberry

### Expected behaviour
I expected a successfull installation

### Actual behaviour
docker pull bitwardenrs/server:raspberry

Error response from daemon: manifest for bitwardenrs/server:raspberry not found: manifest unknown: manifest unknown

**Comment by BlackDex** [comment]
Please read the following:
[[NOTE] Update to the latest version before reporting an issue](https://github.com/dani-garcia/bitwarden_rs/issues/1180)
[Which container image to use](https://github.com/dani-garcia/bitwarden_rs/wiki/Which-container-image-to-use)
